src/en/lilymanga/src/eu/kanade/tachiyomi/extension/en/lilymanga/LilyManga.kt

```diff
@@ -1,9 +1,23 @@
 package eu.kanade.tachiyomi.extension.en.lilymanga
 
 import eu.kanade.tachiyomi.multisrc.madara.Madara
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-class LilyManga : Madara("Lily Manga", "https://lilymanga.net", "en", SimpleDateFormat("yyyy-MM-dd", Locale.US)) {
+class LilyManga : Madara(
+    "Lily Manga",
+    "https://lilymanga.net",
+    "en",
+    dateFormat = SimpleDateFormat("yyyy-MM-dd", Locale.US),
+) {
+    override val client = super.client.newBuilder()
+        .rateLimit(1, 2)
```

Comment:
```suggestion
        .rateLimitHost(baseUrl.toHttpUrl(), 1, 2)
```

I don't think the CDN affects rate limiting. (requires import)